Form-Controls/index.html

```diff
@@ -1,28 +1,101 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form action="results.html" method="get" id="form-info">
+        <div class="form-control">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            placeholder="username"
+          />
+        </div>
+        <div class="form-control">
+          <label for="name">Email:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email"
+            required
+          />
+        </div>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+        <div class="form-control" id="colour-radio">
+          <label>Colour:</label>
+          <div>
+            <label for="radio1" style="color: red">Red</label>
+            <input type="radio" name="colour" id="radio1" value="red" />
+          </div>
+          <div>
+            <label for="radio2" style="color: rgba(248, 244, 5, 0.907)"
+              >Yellow</label
+            >
+            <input type="radio" name="colour" id="radio2" value="yellow" />
+          </div>
+          <div>
+            <label for="radio3" style="color: rgb(253, 8, 200)">Pink</label>
+            <input type="radio" name="colour" id="radio3" value="pink" />
+          </div>
+        </div>
+        <div class="form-control">
+          <label for="size">size:</label>
+          <select name="size" id="size">
+            <option value="xs" label="XS">XS</option>
+            <option value="s" label="S">S</option>
+            <option value="m" label="M">M</option>
+            <option value="l" label="L">L</option>
+            <option value="xl" label="XL">XL</option>
+            <option value="xxl" label="XXL">XXL</option>
+          </select>
+        </div>
+        <div class="form-control">
+          <label for="DeliveryDate">Delivery Date:</label>
+          <input
+            type="date"
+            name="date"
+            id="DeliveryDate"
+            min="2024-10-19"
+            max="2024-11-08"
+          />
+        </div>
+        <div class="form-control">
```

Comment:
Did the acceptance criteria ask for a phone number?

Reply:
Nope, I deleted this tag as well.